pom.xml

```diff
@@ -182,16 +183,30 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- For GlobalMatrixAuthorizationStrategy -->
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>matrix-auth</artifactId>
+            <version>1.1</version>
+            <scope>test</scope>
+        </dependency>
+
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
-            <artifactId>workflow-job</artifactId>
+            <artifactId>workflow-cps</artifactId>
             <version>${workflow.version}</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jenkins-ci.ui</groupId>
+                    <artifactId>jquery-detached</artifactId>
```

Comment:
Looks like a workaround for your use of an ancient (1.x) version.

Reply:
This was previously suggested by @batmat I think, I originally pinned a version, but yes, this is being added as a work around.

Reply:
Suggest leaving a `TODO` comment requesting this be deleted as soon as the relevant `workflow-*` dependencies are upgraded to something not so ridiculously old.

Reply:
TODO Added